files/classes/base.py

```diff
@@ -45,3 +46,80 @@ def age_timedelta(self) -> timedelta:
 	@property
 	def age_string(self) -> str:
 		return format_age(self.created_utc)
+
+
+class CreatedDateTimeBase(Base):
```

Comment:
consider just using `CreatedBase` for this purpose rather than creating a new class. we should probably be migrating them all at once